tests/client/test_client_task_meilisearch.py

```diff
@@ -91,3 +92,30 @@ def test_get_task_inexistent(client):
     """Tests getting a task that does not exists."""
     with pytest.raises(Exception):
         client.get_task("abc")
+
+
+def test_cancel_tasks(client):
+    """Tests cancel a task with uid 1."""
+    task = client.cancel_tasks({"uids": ["1"]})
```

Comment:
```suggestion
    task = client.cancel_tasks({"uids": ["1", "2"]})
```

To ensure you parse the List correctly